apps/web/src/lang/fr.json

```
@@ -5552,6 +5580,10 @@
     "defaultMessage": "Toute langue (anglais ou français)",
     "description": "No preference for language ability - will accept English or French"
   },
+  "YyT0ua": {
+    "defaultMessage": "Publier une annonce",
+    "description": "Title to publish a process' advertisement"
```

Is this what is meant in the description? Will need to change the original English string and id if so.
```suggestion
    "description": "Title to publish an advertisement of a process"
```